code/datums/ai/idle_behaviors/jitterskull_search.dm

```diff
@@ -0,0 +1,41 @@
+// Idle visuals and SFX when jitterskull has no target; complements find_target without replacing it
+
+/datum/idle_behavior/jitterskull_search
+	/// Minimum time without a target before enabling search visuals
+	var/search_debounce = 30
+	/// Cooldown for announcing search text
+	var/announce_cooldown = 50
+
+/datum/idle_behavior/jitterskull_search/perform_idle_behavior(delta_time, datum/ai_controller/controller)
+	. = ..()
+	var/mob/living/simple_animal/hostile/rogue/jitterskull/J = controller.pawn
+	if(!istype(J))
+		return
+	if(J.is_stalking || J.is_guarding || J.is_feasting || J.is_dying || J.stat == DEAD)
+		return
+	// Occasional idle chatter
+	if(world.time >= J.next_idle_chatter_time)
+		if(prob(60))
+			var/chatter = pick('sound/mobs/jitter_chatter1.ogg','sound/mobs/jitter_chatter2.ogg','sound/mobs/jitter_chatter3.ogg','sound/mobs/jitter_chatter4.ogg')
+			playsound(J, chatter, 50, FALSE)
+		J.next_idle_chatter_time = world.time + rand(30, 80)
+	var/atom/current = controller.blackboard[BB_BASIC_MOB_CURRENT_TARGET]
+	// If we have a valid target, ensure normal alpha and reset timers
+	if(current && !QDELETED(current) && !(isliving(current) && current:stat == DEAD))
```

Comment:
`current && !QDELETED(current)` is redundant, also we don't use `:`, you should cast to a `/mob/living` var and do a typecheck